[13.0][ADD] account_invoice_show_currency_rate: Show currency rate in invoices (#968)
OCA/account-invoicing: account_invoice_show_currency_rate/README.rst

    :target: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_show_currency_rate
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/95/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

This module shows the currency rate applied in invoices, so you can visually verify what is going to be applied for the exchange, or which one was applied once converted to company currency.

**Table of contents**

.. contents::
   :local:

Configuration
=============

Enable the option for multiple currencies in your instance:

#. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies
#. Go to any draft invoice
#. Change the invoice currency
#. The proper currency rate, based on the invoice date and the selected currency, will be shown.
#. Add any invoice line.
#. Odoo has already generated the journal item lines with the rate applied, so the currency rate shown is get from the division between the amount in currency by the amount in company currency.

> It's really necessary to exists some rates about not company currency distinct to 1.0, for example from EUR:

Reviewer: ```suggestion
Some rates must be defined (and be distinct to 1.0) for currencies different from the company default.
```

   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
    :target: https://github.com/OCA/account-invoicing/tree/13.0/account_invoice_show_currency_rate
    :alt: OCA/account-invoicing
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_show_currency_rate
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/95/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module show currency rate applied in invoices.

Reviewer 2: ```suggestion
This module shows the currency rate applied in invoices, so you can visually verify what is going to be applied for the exchange, or which one was applied once converted to company currency.
```

**Table of contents**

.. contents::
   :local:

Usage
=====

> o use this module, the company have to access rights Multi Currencies.

Reviewer 2: I agree this should be in the configuration section, and with this rephrasing. Having steps is for differentiating the things you do on screen:
```suggestion
Enable the option for multiple currencies in your instance:

#. Go to *Invoicing > Configuration > Settings*
#. Search for or locate the section "Currencies".
#. Enable "Multi-Currencies" option there.
```

> o use this module, the company have to access rights Multi Currencies.
> 
> #. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies

Reviewer: This would probably fit better under `Configuration`. Also, maybe it could be clearer like this:
```suggestion
To use this module, the company must have Multi Currencies enabled.

#. Go to Invoicing > Configuration > Settings > Currencies > Multi-Currencies
```

> #. Go to any draft invoice

Reviewer 2: Please always put whole steps. How do you get to any draft invoice? Do these instructions creating a new invoice. I'll explain later why.

#. Change the invoice currency

> #. A default currency rate, based on invoice date and the selected currency will be set

Reviewer 2: ```suggestion
#. The proper currency rate, based on the invoice date and the selected currency, will be shown.
```

> 

Reviewer 2: Add now usage instructions when the invoice already contains lines and there are totals to the division computation:

```suggestion
#. Add any invoice line.
#. Odoo has already generated the journal item lines with the rate applied, so the currency rate shown is get from the division between the amount in currency by the amount in company currency.

```

Bug Tracker
===========

Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-invoicing/issues>`_.
In case of trouble, please check there if your issue has already been reported.
If you spotted it first, help us smashing it by providing a detailed and welcomed
`feedback <https://github.com/OCA/account-invoicing/issues/new?body=module:%20account_invoice_show_currency_rate%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.

Do not contact contributors directly about support or help with technical issues.

Credits
=======

Authors
~~~~~~~

* Tecnativa

Contributors
~~~~~~~~~~~~

* `Tecnativa <https://www.tecnativa.com>`_:

>   * Víctor Martínez

Reviewer 2: Designer also counts here :wink:
